d2core/d2input/ebiteninput.go

```diff
@@ -0,0 +1,226 @@
+package d2input
+
+import (
+	"github.com/hajimehoshi/ebiten"
+	"github.com/hajimehoshi/ebiten/inpututil"
+)
+
+func ebitenInput(im *inputManager) error {
+	im.key[Key0]				= int(ebiten.Key0)
```

Comment:
Package is still `d2input`, so `ebiten` dependency is still here.

Reply:
was i supposed to make a new directory?

Reply:
As I mentioned earlier, it doesn't matter how many or few files you have in a package. This change just moves the ebiten reference to a different file, and does not actually remove it altogether. You would have to create a new package underneath d2input (for example d2inputebiten), and put the ebiten bits in there.